cmd/alibabaosstarget-adapter/main.go

```diff
@@ -22,5 +22,5 @@ import (
 )
 
 func main() {
-	pkgadapter.Main("alibabaosstarget-adapter", alibabaosstarget.EnvAccessorCtor, alibabaosstarget.NewTarget)
+	pkgadapter.Main("alibabaosstarget", alibabaosstarget.EnvAccessorCtor, alibabaosstarget.NewTarget)
```

Comment:
I fully endorse this 👍
We had a lot of inconsistencies regarding whether the "-adapter" suffix was included so this is a nice touch IMO.